src/diannes-southwest-salad.md

```diff
@@ -0,0 +1,29 @@
+# Dianne's Southwestern Cornbread Salad
+
+![Dianne's Southwestern Cornbread Salad](pix/diannes-cornbread-salad.webp)
+
+Dianne's Southwestern Cornbread Salad is as colorful as it is delicious.
```

Comment:
Maybe elaborate in the description. Talk about the history of the salad or a bit of background

Reply:
There's not really much description to it. It's just a random salad that my mom made occasionally. It's just from a scrap of a cookbook page. There's plenty of other recipes that have a minimal description or none at all. I just wrote down the caption from the cookbook page. If you want I can just get rid of the description. I don't care.

Reply:
K